### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by orionstardust**

### CODE REVIEW

File: contracts/shared/libraries/LibERC1155.sol
Comment (cinnabarhorse) [inline contracts/shared/libraries/LibERC1155.sol:1]: Removing this because it's defined elsewhere? @orionstardust

Comment (orionstardust) [reply contracts/shared/libraries/LibERC1155.sol:1]: @cinnabarhorse Same event is defined in LibERC721. Duplication made conflicts in diamond ABI.

Comment (orionstardust) [reply contracts/shared/libraries/LibERC1155.sol:1]: @cinnabarhorse Regarding this event, This event is called in `setApprovalForAll()` function only. This function is in `AavegotchiFacet` alrerady. Do we need to move this function and event?

Comment (cinnabarhorse) [reply contracts/shared/libraries/LibERC1155.sol:1]: You can remove it if it's not being called

**Pull request merged**